### ISSUE TITLE
bug(a11y): pagination component has insufficient contrast (3.11) when using "dark" theme.

### ISSUE THREAD

**Issue by miclgael** [miclgael/issues.michaelgale.dev#6]
## Steps to reproduce the issue

1. Visit https://www.michaelgale.dev/blog
2. Scroll down until pagination component is in view

![screenshot of the affected component](https://github.com/miclgael/issues/assets/4301358/6dd6c634-dc61-4263-a6b3-70308522d4fa)

## Expected result

- First, Prev, Next and Last buttons will have sufficient contrast.

## Actual result

- Add a dark variant of `var(--gray-dark)`, which is slightly _less_ dark

**Comment by miclgael** [comment]
Fixed in release `v3.5.22`
